gradle.properties

```diff
@@ -1,3 +1,3 @@
 # suppress inspection "UnusedProperty" for whole file
 group = com.twcable.gradle
-version = 2.0.2
+version = 2.0.3
```

Comment:
Can you please remove the version number change? I like to have a separate commit specifically for that where the RELEASE_NOTES file is updated, the commit is tagged, and the new release is sent to Bintray. (See for example d4bcddfb2f1fba7abe44e62ba34546ac42472d2b )